wpimath/src/main/java/edu/wpi/first/wpilibj/controller/SimpleMotorFeedforward.java

```diff
@@ -47,6 +51,24 @@ public double calculate(double velocity, double acceleration) {
     return ks * Math.signum(velocity) + kv * velocity + ka * acceleration;
   }
 
+  /**
+   * Calculates the feedforward from the gains and setpoints.
+   *
+   * @param currentVelocity The current velocity setpoint.
+   * @param nextVelocity The next velocity setpoint.
+   * @param dtSeconds Time between velocity setpoints in seconds.
+   * @return The computed feedforward.
+   */
+  public double calculate(double currentVelocity, double nextVelocity, double dtSeconds) {
+    var plant = LinearSystemId.identifyVelocitySystem(this.kv, this.ka);
+    var feedforward = new LinearPlantInversionFeedforward<>(plant, dtSeconds);
```

Comment:
I'm curious, what's the performance impact of instantiating these objects on every control loop iteration?

Reply:
Good question; I don't know. We do something similar in ProfiledPIDController when instantiating a TrapezoidProfile every loop to do the calculation for that timestep.